libhb/handbrake/qsv_libav.h

```diff
@@ -158,7 +158,9 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define HB_QSV_ID_BUFFER MFX_MAKEFOURCC('B','U','F','F')
 #define HB_QSV_ID_FRAME  MFX_MAKEFOURCC('F','R','M','E')
 
-#define HB_QSV_SURFACE_NUM              80
+// You can blame HEVC encoding as it has a maximum gop size of 256.
+#define HB_QSV_SURFACE_NUM              268
```

Comment:
Main setting which defines number of frames is a number of B frames (GopRefDist), not GopSize